### PR TITLE
PS-7141 Update innodb_show_status

### PR DESCRIPTION
**Pull request by patrickbirch**
modified innodb_show_status
added note

### CODE REVIEW

**Review by kathy-williamson — APPROVED** [review]

File: doc/source/diagnostics/innodb_show_status.rst [hunk 1]
```diff
@@ -386,6 +386,10 @@ the system.
    :column READ_VIEW_UPPER_LIMIT_TRX_ID: This is the highest transactions ID at the time the view was created. This means that it should not see newer transactions with IDs bigger than or equal to that value.
    :column READ_VIEW_LOW_LIMIT_TRX_ID: This is the latest committed transaction ID at the time the oldest view was created. This means that it should see all transactions with IDs smaller than or equal to that value.
 
+.. note::
+
+    Starting with |Percona Server| 8.0.20-11, the data type for the ``READ_VIEW_LOW_LIMIT_TRX_NUMBER``, ``READ_VIEW_UPPER_LIMIT_TRX_ID``, and the ``READ_VIWE_LOW_LIMIT_TRX_ID`` columns in the ``XTRADB_READ_VIEW`` are changed from ``VARCHAR(18)`` to ``BIGINT UNSIGNED``. The columns contain 64-bit integers, which are too large for ``VARCHAR(18)``.
```
Comment (percona-ysorokin) [inline doc/source/diagnostics/innodb_show_status.rst]: @patrickbirch 
Shouldn't this be
... the data type for ... **IS** changed...
Although I could be wrong, English is not my native language :)

**Review by percona-ysorokin — APPROVED** [review]
LGTM
with one small grammar-related comment

File: doc/source/diagnostics/innodb_show_status.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/diagnostics/innodb_show_status.rst]: Rewrote the note to make it easier to parse.

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**